### ISSUE TITLE
Use default aws credentials 

### ISSUE THREAD

**Issue by rvdwijngaard** [elbuo8/sqs.js#5]
Hi, 

I have been checking out your library and it seems to fit quite well with my requirements; I have a question: currently you have to set the aws credentials explicitly with a config object; It would be nice if this is optional and that the aws-sdk uses the default credentials.

Regards,
Ron

**Comment by elbuo8** [comment]
You can pass in an already built `SQS` object. Would that be cool?

**Comment by rvdwijngaard** [comment]
Don't think that is helpfull; i would suggest that the aws settings are not mandatory, at least the keys. Aws-sdk has its own way of determining the proper credentials. 

I like the way how ryanfitz solved it with vogels. https://github.com/ryanfitz/vogels. This module exposes the aws sdk as a property so you can configure aws to whathever your needs are.

**Comment by elbuo8** [comment]
The sqs settings are not mandatory if you pass in the built in sqs object.

**Comment by rvdwijngaard** [comment]
That's true but I like the abstraction of your module. If the aws keys are not mandatory it works with the standard aws config.
